controllers/sriovnetworkpoolconfig_controller.go

```diff
@@ -76,9 +76,13 @@ func (r *SriovNetworkPoolConfigReconciler) Reconcile(ctx context.Context, req ct
 				return reconcile.Result{}, err
 			}
 		}
-		if utils.ClusterType == utils.ClusterTypeOpenshift && !isHypershift {
-			if err = r.syncOvsHardwareOffloadMachineConfigs(instance, false); err != nil {
-				return reconcile.Result{}, err
+		if utils.ClusterType == utils.ClusterTypeOpenshift {
+			if !isHypershift {
+				if err = r.syncOvsHardwareOffloadMachineConfigs(instance, false); err != nil {
+					return reconcile.Result{}, err
+				}
+			} else {
+				logger.Info("hwol not yet implemented on hypershift")
```

Comment:
Can you make this more severe? I'm thinking:

`
Logger.Warn("Ignoring request to enable HWOL (running on Hypershift)")
`